File: javascript/easy/raindrops.js
export const convert = (num) => {
  let string = "";
  if (num % 3 === 0) {
    string += "Pling";
  }
  if (num % 5 === 0) {
    string += "Plang";
  }
  if (num % 7 === 0) {
    string += "Plong";
  }
  if (!string) {
    string = num.toString();
  }
  return string;
};

File: python/raindrops.py
def convert(number):
    string = ""
    if not number % 3:
        string += "Pling"
    if not number % 5:
        string += "Plang"
    if not number % 7:
        string += "Plong"
    if not string:
        string += str(number)
    return string
